services/basic_types.proto

```diff
@@ -156,6 +156,17 @@ message TransactionID {
      * Whether the Transaction is of type Scheduled or no
      */
     bool scheduled = 3;
+
+    /**
+     * The unique id of a "child" transaction that was executed on behalf of 
```

Comment:
I don't think we should mention parent/child when talking about nonce, except maybe in detailing a particular example of a nonce. Instead we should just state that this is an opaque ID used to track internal transactions that are spawned from user submitted transactions.